pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -73,6 +74,7 @@ func (admitter *KubeVirtUpdateAdmitter) Admit(ar *admissionv1.AdmissionReview) *
 
 	results = append(results, validateCustomizeComponents(newKV.Spec.CustomizeComponents)...)
 	results = append(results, validateCertificates(newKV.Spec.CertificateRotationStrategy.SelfSigned)...)
+	results = append(results, validateTLSConfiguration(newKV.Spec.Configuration.TLSConfiguration)...)
```

Comment:
Can we make the validation conditional based on a comparison between the `curr` and `new` KV configs?

Reply:
Thanks! Done!